webserver/views/datasets.py

```diff
@@ -92,6 +93,41 @@ def view(id):
     )
 
 
+def get_json(id):
+    dataset = get_dataset(id)
+    dataset_clean = {
+        "name": dataset["name"],
+        "description": dataset["description"],
+        "classes": [],
+        "public": dataset["public"],
+    }
+    for cls in dataset["classes"]:
+        dataset_clean["classes"].append({
+            "name": cls["name"],
+            "description": cls["description"],
+            "recordings": cls["recordings"],
+        })
+
+    return dataset_clean
+
+CSV_DIR = "./code/data/files/csv_files"
+
+@datasets_bp.route("/<uuid:dataset_id>/download")
```

Comment:
same with the text in the template, let's have a name here which indicates that we're just downloading the dataset schema